hack/argoproj.io_applicationsets.yaml

```diff
@@ -0,0 +1,6377 @@
+
```

Comment:
Taken from https://github.com/argoproj-labs/applicationset/blob/v0.2.0/manifests/crds/argoproj.io_applicationsets.yaml as it is needed for our tests.